### PR TITLE
Example update: Getting started with BitOps v2.0 - Terraform

### PR DESCRIPTION
**Pull request by ccapell**
# Description

Added example for Blogpost
Updated .gitignore
- added .DS_Store
- Added .vscode/settings.json

### CODE REVIEW

**Review by armab — CHANGES_REQUESTED** [review]
Could you add some `README.md` for this example?

See other https://github.com/bitovi/bitops/tree/main/docs/examples for inspiration

**Comment by ccapell** [comment]
Added readme

File: docs/examples/bitops2-terraform-blog/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Bitops Operations Repo
+
+Welcome to Bitops! This serves as a starting point for deploying your application to the cloud.
+
+This repo can be run as is with
```
Comment (armab) [inline docs/examples/bitops2-terraform-blog/README.md:5]: Looks similar to https://github.com/bitovi/bitops/tree/main/docs/examples/bitops-deploy-heyemoji/README.md

Could we add some specifics what's going on in this example and how it's different from others?
https://github.com/bitovi/bitops/tree/main/docs/examples/terraform-and-ansible

Comment (ccapell) [reply docs/examples/bitops2-terraform-blog/README.md:5]: We can update it after the blog post goes live with the URL to the blog post.  This is the example for the blog post.  I need to include the link to this repo BEFORE the blog post goes live.  Until the blog goes live we won't have the URL.  See the issue?  I did add a place holder though.

File: docs/examples/bitops2-terraform-blog/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Bitops Operations Repo
+
+Welcome to Bitops! This serves as a starting point for deploying your application to the cloud.
```
Comment (armab) [inline docs/examples/bitops2-terraform-blog/README.md:3]:
```suggestion
This example shows a minimal Terraform configuration that creates an EKS cluster in `test` environment with a shared `_default` dir feature as an introduction to BitOps v2.0.
```

File: docs/examples/bitops2-terraform-blog/README.md (hunk 1 above)
Comment (armab) [reply docs/examples/bitops2-terraform-blog/README.md:5]: I think having a very small brief description of what this example does would be sufficient without failing into the chicken vs egg problem with URL.

Just added a specific code suggestion here: https://github.com/bitovi/bitops/pull/299#discussion_r966404418

**Review by armab — APPROVED** [review]

**Review by PhillypHenning — COMMENTED** [review]
Looks good to me

**Pull request merged**